scipy/linalg/decomp.py

```diff
@@ -1267,3 +1267,138 @@ def hessenberg(a, calc_q=False, overwrite_a=False, check_finite=True):
     q, info = orghr(a=hq, tau=tau, lo=lo, hi=hi, lwork=lwork, overwrite_a=1)
     _check_info(info, 'orghr (hessenberg)', positive=False)
     return h, q
+
+
+def cdf2rdf(w, v):
```

Comment:
You have to add the function name to `__all__` at the top so that the test suite can import it.

Reply:
this was done

Reply:
Yes, the review didn't sweep this because I've commented on the this line instead of the line at the top.